src/databrowser.js

```diff
@@ -32,7 +32,7 @@ define([
             head = pane.find(".fd-head-external-sources");
             headHeight = head.outerHeight(true) || 0;
             pane.data("min-size", headHeight)
-                .height(tree.height() * DATABROWSER_HEIGHT)
+                .height(Math.min(tree.height() * DATABROWSER_HEIGHT, headHeight * 12))
```

Comment:
That's correct. It's not the screen size though. It's the tree height. So, it's window size - browser toolbar - HQ header - breadcrumbs - vellum header - HQ footer

It looked ok locally with small breadcrumbs (on 1366px), but may change again depending on the coming UI changes